.lando.yml

```
@@ -4,9 +4,21 @@ config:
   php: '7.4'
   via: apache:2.4
   webroot: web
-  database: mariadb:10.4
   xdebug: false
 services:
+  database:
+    type: compose
+    services:
+      image: mariadb:latest
+      command: docker-entrypoint.sh mariadbd
+      restart: always
+      ports:
+        - '3306'
+      environment:
+        MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 'true'
+        MARIADB_DATABASE: drupal9
```

I found that I was able to use `db-export` and `db-import` after adding this and running `lando rebuild`
```suggestion
        MARIADB_DATABASE: drupal9
        MYSQL_DATABASE: drupal9
```